python/cudf/tests/test_replace.py

```diff
@@ -187,3 +187,10 @@ def test_fillna_dataframe(fill_type, inplace):
         got = gdf
 
     assert_eq(expect, got)
+
+def test_series_fillna_dtype():
+    df = DataFrame()
+    df['a'] = [1, 2, 3]
```

Comment:
Should probably have a null value in here just to confirm it fills correctly as well 😄

Reply:
good idea :P definitely going to be adding a few more tests